catroid/src/main/java/org/catrobat/catroid/common/defaultprojectcreators/JumpingSumoProjectCreator.java

```diff
@@ -65,35 +67,36 @@ public Project createDefaultProject(String projectName, Context context, boolean
 			throw new IllegalArgumentException("Project with name '" + projectName + "' already exists!");
 		}
 
+		Project defaultJumpingSumoProject = new Project(context, projectName, landscapeMode);
+		defaultJumpingSumoProject.setDeviceData(context); // density anywhere here
+		XstreamSerializer.getInstance().saveProject(defaultJumpingSumoProject);
+		ProjectManager.getInstance().setProject(defaultJumpingSumoProject);
+
+		File sceneDir = defaultJumpingSumoProject.getDefaultScene().getDirectory();
+		File imageDir = new File(sceneDir, IMAGE_DIRECTORY_NAME);
+
 		double landscapePortraitFactor = 1.63;
 
-		String backgroundName = context.getString(R.string.add_look_drone_video);
+		String backgroundName = context.getString(R.string.add_look_jumping_sumo_video);
 
 		SetSizeToBrick setSizeBrick = new SetSizeToBrick(60.0);
 
 		Script whenProjectStartsScript = new StartScript();
 		whenProjectStartsScript.addBrick(setSizeBrick);
 
 		Script whenSpriteTappedScript = new WhenScript();
-		Project defaultJumpingSumoProject = new Project(context, projectName, true);
-		File sceneDir = defaultJumpingSumoProject.getDefaultScene().getDirectory();
-
-		defaultJumpingSumoProject.setDeviceData(context); // density anywhere here
-		XstreamSerializer.getInstance().saveProject(defaultJumpingSumoProject);
-		ProjectManager.getInstance().setProject(defaultJumpingSumoProject);
 
 		backgroundImageScaleFactor = ImageEditing.calculateScaleFactorToScreenSize(
 				R.drawable.drone_project_background, context);
 
-		File backgroundFile = ResourceImporter.createImageFileFromResourcesInDirectory(context.getResources(), R.drawable.ic_video, sceneDir,
+		File backgroundFile = ResourceImporter.createImageFileFromResourcesInDirectory(context.getResources(), R
+						.drawable.ic_video, imageDir,
```

Comment:
as for the AR drone please fix the formatting issues in this file

Reply:
done.